### PR TITLE
Redirecting haproxy logs

### PR DESCRIPTION
**Pull request by amitbose327**

### CODE REVIEW

**Comment by ankanm** [comment]
LGTM

File: ansible/roles/bamboo/handlers/main.yml [hunk 1]
```diff
@@ -3,3 +3,10 @@
   become: yes
   tags:
    - bamboo
+
+- name: restart rsylog
+  service: name=rsyslog state=restarted
+  become: yes
+  tags:
+   - ryslog
```
Comment (jhulten) [inline ansible/roles/bamboo/handlers/main.yml]: Is this tag correct?

File: ansible/roles/bamboo/templates/haproxy_logrotate.j2 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{ haproxy_log_dir }}/haproxy.log {
+    daily
+    rotate 5
+    missingok
+    notifempty
+    compress
+    delaycompress
+    postrotate
+        invoke-rc.d rsyslog rotate >/dev/null 2>&1 || true
```
Comment (jhulten) [inline ansible/roles/bamboo/templates/haproxy_logrotate.j2]: `|| true`? Do we want this to always succeed?

Comment (amitbose327) [inline ansible/roles/bamboo/templates/haproxy_logrotate.j2]: This is the logrotate.d config file that comes in when haproxy pkg gets installed. I just updated the log dir and the rotate from 52 to 5.

Comment (jhulten) [inline ansible/roles/bamboo/templates/haproxy_logrotate.j2]: So changed from 52 weeks to 5 days.

Comment (amitbose327) [inline ansible/roles/bamboo/templates/haproxy_logrotate.j2]: It was 52 days earlier. Changed it to 5 days.

**Pull request merged**